src/SDKs/Batch/DataPlane/Azure.Batch/GeneratedProtocol/Models/JobScheduleAddParameter.cs

```diff
@@ -46,6 +47,7 @@ public JobScheduleAddParameter()
         /// the schedule as metadata.</param>
         public JobScheduleAddParameter(string id, Schedule schedule, JobSpecification jobSpecification, string displayName = default(string), IList<MetadataItem> metadata = default(IList<MetadataItem>))
         {
+            JobSpecification = new JobSpecification();
```

Comment:
@matthchr won't this just be overridden a few lines down by the following:

```cs
JobSpecification = jobSpecification;
```

Reply:
Also generated by AutoRest, I don't have any control over this =(